Dockerfile

```diff
@@ -18,27 +19,24 @@ COPY . .
 RUN pip3 install --no-deps .
 
 
-FROM ${registry}/ncigdc/python38-httpd:${base_version}
+FROM ${REGISTRY}/ncigdc/python3.8-httpd:${BASE_VERSION}
+ARG NAME
 
 LABEL org.opencontainers.image.title=${NAME} \
       org.opencontainers.image.description="${NAME} container image" \
       org.opencontainers.image.source="https://github.com/NCI-GDC/${NAME}" \
       org.opencontainers.image.vendor="NCI GDC"
 
+RUN dnf install -y libpq-15.0
 
-RUN mkdir -p /var/www/indexd/ \
-  && chmod 777 /var/www/indexd \
-  && a2dissite 000-default
+RUN mkdir -p /var/www/${NAME}/ \
+  && chmod 777 /var/www/${NAME}
 
-COPY wsgi.py /var/www/indexd/
-COPY bin/indexd /var/www/indexd/
-COPY --from=build /usr/local/lib/python3.8/dist-packages /usr/local/lib/python3.8/dist-packages
+COPY wsgi.py /var/www/${NAME}/
+COPY bin/${NAME} /var/www/${NAME}/
+COPY --from=build /venv/lib/python3.8/site-packages /venv/lib/python3.8/site-packages
```

Comment:
Random comment: I see this pattern in most of the python dockers. I came up with a different solution: https://github.com/NCI-GDC/bio-python-template/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/Dockerfile#L19 . Essentially create the wheel in the builder image and copy just that to the final image for installation

Reply:
your approach seems better. 
I will update our team and probably will use your approach. Thanks for sharing.

Reply:
I like the idea of using the wheel. To better separate concerns, should the build publish the wheel and the next phase download instead of using a copy from the previous phase? (this is a question and not guidance)

Reply:
>should the build publish the wheel and the next phase download instead of using a copy from the previous phase?

The problem with this approach is that we won't be able to build docker locally directly. 
We have to build wheel and push it to nexus first.